grails-app/domain/org/pih/warehouse/order/OrderAdjustment.groovy

```diff
@@ -180,6 +180,11 @@ class OrderAdjustment implements Serializable, Comparable<OrderAdjustment> {
             return hasPrepaymentInvoice && !hasRegularInvoice && order.placed && !invoicedFullAmount
         }
 
+        // check if by any chance it was invoiced as cancelled
```

Comment:
I think the if check below makes sense, but I am not sure if the comment here makes sense. If it was invoiced as canceled then it will have a regular invoice. Plus looking at Katarzyna's comment in the ticket, there was a case about the total adjustments being 0, this does not mean it was canceled. Am I missing something here?